src/main/scala/nl.knaw.dans.easy.authinfo/CommandLineOptions.scala

```diff
@@ -28,8 +28,9 @@ class CommandLineOptions(args: Array[String], configuration: Configuration) exte
   val description: String = s"""Provides consolidated authorization info about items in a bag store."""
   val synopsis: String =
     s"""
-       |  $printedName run-service # Runs the program as a service
-       |  $printedName <item-id> # Retrieves the information from the cache or directly from the bag-store""".stripMargin
+       |  $printedName get <item-id> # Retrieves the information from the cache or directly from the bag-store
+       |  $printedName delete <solr-query> # Delete the selected item(s) from the SOLR index
```

Comment:
It would be nice to have an example of a solr-query here

Reply:
I know, but I was hoping no one would notice 😉
I'm not so proficient with SOLR. The oly query I know is `"*"` to delete everything. But I'll see what I can do!